packages/react/src/utils/listbox-option.ts

```diff
@@ -0,0 +1,19 @@
+interface ListboxOption {
+    disabled?: boolean;
+    value: string;
+    label?: string;
+    caption?: string;
+}
+
+type Value = string | string[];
+
+export function findOptionsByValue<T extends ListboxOption>(
```

Comment:
Soit on share les interfaces avec un fichier (ex: table/types.ts), sinon ya pas de raison de mettre tout l'interface au complet si t'a juste besoin de value dans cette function. Le extends ne defini que le minimum requis, l'object peut avoir d'autre prop supplementaire.

```suggestion
export function findOptionsByValue<T extends { value: string }>(
```